docs/index.rst

```diff
@@ -53,3 +53,5 @@ The opinions and proposals stated here are merely related to the authors' experi
    getting_started/index.rst
    guidelines/index.rst
    use-cases/index.rst
+   docker/index.rst
```

Comment:
I choose to create a new section because it's easier to find and normally you would set up a container and then check the uses cases to find your specific use case you want to apply inside the container.